st2api/tests/base.py

```diff
@@ -63,6 +63,8 @@ class APIControllerWithRBACTestCase(FunctionalTest, CleanDbTestCase):
     Base test case class for testing API controllers with RBAC enabled.
     """
 
+    enable_auth = True
```

Comment:
The variable is used in FunctionalTest._do_setUpClass few lines highier. It mocks auth config option.